core/models/instance.py

```diff
@@ -206,10 +209,10 @@ def get_last_history(self):
         # TODO: Profile Option
         # except InstanceStatusHistory.DoesNotExist:
         # TODO: Profile current choice
-        last_history = self.instancestatushistory_set.all().order_by(
-            '-start_date')
+        last_history = self.instancestatushistory_set.order_by(
+            '-start_date').first()
```

Comment:
What is `self.instancestatushistory_set.order_by('-start_date')` returns `None`?  Is `instancestatushistory_set` or `order_by` guarantee to return an empty collection?